drivers/clock_control/Kconfig.numaker

```diff
@@ -0,0 +1,10 @@
+# NuMaker clock controller driver configuration options
+
+# Copyright (c) 2022 Nuvoton Technology Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config CLOCK_CONTROL_NUMAKER_SCC
+	bool "NuMaker system clock controller driver"
+	depends on SOC_FAMILY_NUMAKER
```

Comment:
DT dependency on `nuvoton_numaker_scc`?

Reply:
Change to depends on DT_HAS_NUVOTON_NUMAKER_SCC_ENABLED